conbench/cli.py

```diff
@@ -81,6 +86,61 @@ def _to_cli_name(name):
     return f"--{name.replace('_', '-')}"
 
 
+def log_timing_report_across_results(results):
+    # Find tags that are common among all results. That makes sense only when
+    # there is more than one result.
+    if len(results) == 1:
+        common_tag_keys = []
+    else:
+        common_tags = set.intersection(*tuple(set(r["tags"].items()) for r in results))
+        common_tag_keys = [t[0] for t in common_tags]
+        log.info("common tag keys across results: %s", common_tag_keys)
+
+    # For each result, assemble a human-readable string which represents the
+    # case/scenario from the tags. Also assemble a human-readable string that
+    # reports the timing info for this scenario.
+    tagstrings = []
+    timestrings = []
+
+    for r in results:
+        tags = r["tags"]
+
+        # That represents the case/senario. Note that `tags` has been enriched
+        # with key/value pairs beyond the case -- filter these out again.
+        tagstring = ", ".join(
+            f"{k}: {tags[k]}" for k in sorted(tags.keys()) if k not in common_tag_keys
+        )
+
+        # Assume that stdev=0 for now means there is only one sample.
+        if r["stats"]["stdev"] == 0:
```

Comment:
This will throw a keyerror if a case failed because `stats` will not be populated (though `error` will). Well, mostly; I know we were working towards accepting both keys for cases where some iterations failed and some didn't, though I forget where we are in that change—I think Jon merged a server change, but I'm not sure if this CLI can actually handle that situation yet. 

In any case, we should probably write this to handle either of `stats` or `error` being missing without erroring out.